test/metabase/sync/sync_metadata/indexes_test.clj

```diff
@@ -21,7 +21,7 @@
       (is (false? (t2/select-one-fn :database_indexed :model/Field (mt/id :table :not-indexed)))))))
 
 (deftest sync-composite-indexed-columns-test
-  (mt/test-drivers (mt/normal-drivers-with-feature :index-info)
+  (mt/test-drivers (disj (mt/normal-drivers-with-feature :index-info) :mongo)
```

Comment:
As implemented this test can only work for `:sql-jdbc` drivers.
```suggestion
  (mt/test-drivers (mt/normal-drivers-with-feature :index-info :sql-jdbc)
```
Does it make sense to move this to `metabase.driver.sql-jdbc-test`?

Reply:
there is no feature called "sql-jdbc", we need to intersect the feature drivers with `(mt/sql-jdbc-drivers)`

Reply:
> Does it make sense to move this to metabase.driver.sql-jdbc-test?
I think it's fine here as we treat tests for mongo is a kind of special case where most mongo tests are defined in its namespace.

Reply:
Yeah fair enough. I guess what I'm trying to get at is the implementation of this test uses `sql-jdbc.conn/db->pooled-connection-spec` and `sql.tx/create-index-sql`, so it is specific to `sql-jdbc` drivers. If bigquery had indexes this would be more of a problem, but I guess it's okay. Ideally there would be a multimethod `tx/create-index!` that had a default implementation for SQL drivers that called `sql.tx/create-index-sql`. This is how it works for `create-db!` anyway. https://github.com/metabase/metabase/blob/772e8eec90b82159aaa406615014e8d571937d5d/test/metabase/test/data/sql_jdbc/load_data.clj#L198

Anyway, it's not related to this PR and probably not worth the ceremony